components/body/Welcome.js

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
think about how we can refactor portions of this component for reusability -- for example, a majority of the screens in the application mockup Figma file contain a similar `<h1/><h2/><hr/>` block. if we extract that body header text into a separate component it can be re-used for other pages in the future!

Reply:
Ready for review:
Headline.js - stores h1, any other jsx (figma file shows h2 & p tag) that will be used as children, and hr
WelcomeCta.js - dynamic icon, estimate text, & button text

Reply:
@achang-tcs ready for review!

Reply:
@lyoon-tcs great job refactoring these components! i really like the strategy where you pass the children for `Headline.js` as props.

one small note - you can consolidate `import` lines from the same dependency into one line e.g. `import { Col, Container, Row } from react-bootstrap`.